pkg/go/graph/graph_edge.go

```diff
@@ -12,6 +12,7 @@ const (
 	RewriteEdge  EdgeType = 1
 	TTUEdge      EdgeType = 2
 	ComputedEdge EdgeType = 3
+	NoCond       string   = "none"
```

Comment:
Why do we need this? Wouldn't inspecting the `conditions` field be enough? If `nil` it has 0 conditions and we don't have unnecessary allocations 🤔